training_record/views.py

```diff
@@ -79,7 +79,17 @@ def list_records_for_review(self, request):
         '''Return all offCampusRecords for admin'''
         queryset = self.filter_queryset(self.get_queryset()).filter(
             off_campus_event__isnull=False,
-        )
+            ).extra(select={
+                'is_status_submitted':
+                f'status={Record.STATUS_SUBMITTED}'}).order_by(
+                    '-is_status_submitted', '-create_time')
+        if request.user.is_department_admin:
+            return self._get_paginated_response(queryset)
```

Comment:
You should place all department-admin-actionable records ahead of all other records, instead of just `STATUS_SUBMITTED`, same for school admin.